autoload/OmniSharp/py.vim

```diff
@@ -0,0 +1,37 @@
+if !(has('python') || has('python3'))
+  finish
+endif
+
+let s:pycmd = has('python3') ? 'python3' : 'python'
+let s:pyfile = has('python3') ? 'py3file' : 'pyfile'
+let s:module_path_added = 0
+let g:omnisharp_python_path = OmniSharp#util#path_join(['python', 'omnisharp'])
+
+let s:save_cpo = &cpo
+set cpo&vim
+
+if exists('*py3eval')
+  let s:pyeval = function('py3eval')
+elseif exists('*pyeval')
+  let s:pyeval = function('pyeval')
+else
+  exec s:pycmd 'import json, vim'
+  function! s:pyeval(e)
+    exec s:pycmd 'vim.command("return " + json.dumps(eval(vim.eval("a:e"))))'
+  endfunction
```

Comment:
I just downloaded a version of vim7.3 to test this - `pyeval` and `py3eval` have been in vim since 7.4. Well, I couldn't get vim 7.3 to work with the `g:OmniSharp#py#eval` patch I suggested.

Shouldn't we just leave this function out, and make 7.4 the minimum required version of vim? The current code uses `pyeval` anyway. It could be useful to output a warning message here, I suppose?

Reply:
I would agree that having 7.4 as a minimum version would be preferable.